### ISSUE TITLE
Benchmark fails to run with no ingester on `Redis 6.2.5`

### ISSUE THREAD

**Issue by mjsML** [pbotros/river#8]
I tried the vanilla install without arrow, it built and installed just fine. now trying to run the benchmark I get the following error:

```
terminate called after throwing an instance of 'cxxopts::invalid_option_format_error'
  what():  Invalid option format ‘redis_hostname,h’
[4]    4161 abort (core dumped)  ./river_benchmark --redis_hostname 127.0.0.1 --batch_size 1 --row_size

```

My redis instance is local:
```
          _.-``__ ''-._                                             
      _.-``    `.  `_.  ''-._           Redis 6.2.5 (00000000/0) 64 bit
  .-`` .-```.  ```\/    _.,_ ''-._                                  
 (    '      ,       .-`  | `,    )     Running in standalone mode
 |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
 |    `-._   `._    /     _.-'    |     PID: 4018
  `-._    `-._  `-./  _.-'    _.-'                                   
 |`-._`-._    `-.__.-'    _.-'_.-'|                                  
 |    `-._`-._        _.-'_.-'    |           https://redis.io       
  `-._    `-._`-.__.-'_.-'    _.-'                                   
 |`-._`-._    `-.__.-'    _.-'_.-'|                                  
 |    `-._`-._        _.-'_.-'    |                                  
  `-._    `-._`-.__.-'_.-'    _.-'                                   
      `-._    `-.__.-'    _.-'                                       
          `-._        _.-'                                           
              `-.__.-'                                               

4018:M 08 Sep 2021 20:37:43.330 # Server initialized
4018:M 08 Sep 2021 20:37:43.330 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
4018:M 08 Sep 2021 20:37:43.353 * Ready to accept connections
```

**Comment by pbotros** [comment]
Thanks @mjsML ! Looks like a quick bug in how the cxxopts were structured. Fix https://github.com/pbotros/river/pull/9 merged to latest master. Try it out and close this issue out if it works for you.

**Comment by mjsML** [comment]
@pbotros I get another error now: 
```
terminate called after throwing an instance of 'cxxopts::option_not_exists_exception'                                                    
  what():  Option ‘row_size’ does not exist                                                                                            
[1]    4006 abort (core dumped)  ./river_benchmark --redis_hostname 127.0.0.1 --batch_size 1 --row_size 128
```
This is with the `-DRIVER_BUILD_INGESTER=1` build with the latest commit [f26d072
](https://github.com/pbotros/river/commit/f26d0728a4c56c92d61feaedc928cb0f39e58e93)

**Comment by mjsML** [comment]
@pbotros I think I figured it out ... I was able to run the benchmarks using `--sample_size` instead of `--row_size`

**Comment by mjsML** [comment]
Very intestine results :)
![image](https://user-images.githubusercontent.com/7131192/132731245-769af3ae-32b3-4161-a16b-74beacc43139.png)
![image](https://user-images.githubusercontent.com/7131192/132731820-eb9d51c8-d6ac-493b-aef9-087974bbce95.png)

**Comment by pbotros** [comment]
Thanks @mjsML for https://github.com/pbotros/river/pull/11 ! Those results look about right. As you can see, batch_size has a large effect on writing/reading throughputs, as higher batch_size means less relative impact on the overhead of communications with redis. Feel free to close this issue out.

**Comment by mjsML** [comment]
closed by #11
